src/components/Typography/Typography.types.ts

```diff
@@ -0,0 +1,76 @@
+import { TooltipProps } from 'antd'
+
+/**
+ * @link https://ant.design/components/typography#copyable
+ */
+export type CopyConfig = {
+
+  /**
+   * The text to copy.
+   */
+  text?: string;
+
+  /**
+   * Function called when the text is copied.
+   */
+  onCopy?: (event?: React.MouseEvent<HTMLDivElement>) => void;
+
+  /**
+   * Custom copy icon(s): [copyIcon, copiedIcon].
```

Comment:
it is not clear if both a single icon and an array are correct values for this property.